app/views/govuk_publishing_components/components/_fieldset.html.erb

```diff
@@ -1,7 +1,12 @@
 <% text = text || yield %>
-<fieldset class="gem-c-fieldset">
-  <legend class="gem-c-fieldset__legend">
-    <%= legend_text %>
-  </legend>
-  <%= text %>
-</fieldset>
+<% legend_text = legend_text || nil %>
+<% if text.present? || legend_text.present? %>
```

Comment:
Is it necessary to check for the existence of the legend_text here? Is there a situation where you'd want the component to render with only the legend_text?